src/parsers/ZillowParser.js

```diff
@@ -27,14 +27,27 @@ export default class ZillowParser {
   }
 
   get rent() {
-    const [rent] = $('.ds-home-details-chip').text().match(priceRegExp()) || [null]
-    return extractNumber(rent)
+    const container = this.document.querySelector('.data-column-container .summary-container')
+    const containerSpanNodes = container.querySelectorAll('span')
+    if (containerSpanNodes.length) {
+      let rentIndex
```

Comment:
Could we use `Array.prototype.findIndex()` instead?

Reply:
I improve this part of code with latest commit. Please, take a look